### PR TITLE
Feature-Search

### PR DESCRIPTION
**Pull request by Shereen789**
The Basic functionality of Search feature has been implemented.

### CODE REVIEW

File: project1/application.py [hunk 1]
```diff
@@ -45,6 +51,25 @@ def login():
         return "Please Login using your Credentials"
 
 
+@app.route("/search")
+def search():
+    return render_template('search.html')
+
+
+@app.route("/sresults", methods=["POST"])
+def sresults():
```
Comment (krajeshkumar11) [inline project1/application.py]: Place both post and get methods in same route, based on request method implement respective block of code using if and else block.

**Comment by krajeshkumar11** [comment]
@Shereen789  good work. Try implementing provided reviews.

File: project1/application.py (hunk 1 above)
Comment (Shereen789) [reply project1/application.py]: @krajeshkumar11 
Sir, the file is updated in the Search-Book Branch. Kindly check now.

Comment (krajeshkumar11) [reply project1/application.py]: @Shereen789 good work can merge with master.

**Comment by krajeshkumar11** [comment]
@Shereen789 good work can merge with master.

**Pull request merged**